docker/mainnet/bitcoin/bitcoin.mainnet.conf

```diff
@@ -0,0 +1,18 @@
+port=8333
+
+rpcbind=0.0.0.0
+rpcport=8332
+
+rpcallowip=172.100.2.100
+rpcallowip=172.100.2.101
+
+txindex=1
```

Comment:
Should we need to add comment here that txindex is needed for lnd?

Reply:
t required by lnd. We can add comments about this txindex.